docs/guides/proofreading.md

```diff
@@ -81,55 +106,46 @@ frame with any instances from previous frames.
 Here's a strategy that works well for fixing **lost** identities:
 
 1. Turn on colors for predicted instances and use a good color palette (as explained above).
-
-2. Turn on track **trails** using the "**Show Trails**" command in the "View" menu. These trails show where instances in each track were in prior frames. You can determine how many prior frames by setting the "**Trail Length**" (also in the "View" menu).
-
+2. Change the "**Trail Length**" to a number greater than zero. These trails show where instances in each track were in prior frames.
 3. Use the keyboard shortcut for the "**Next Track Spawn Frame**" command in the "Go" menu to jump to frames where a new track identity is spawned.
-
-4. Select the instance with the new track identity—either use the mouse, type a number key to jump to that instance, or use the **`** key to cycle through instances.
-
+4. Select the instance with the new track identity—either use the mouse, type a number key to jump to that instance, or use the **\`** key to cycle through instances.
 5. The color of the track trail may help you determine which track identity should have been used.
-
 6. Hold down the **Control** key (or **Command** key on a Mac) with an instance already selected and you'll see a color-coded list of numbered tracks, like so:
 
-|track-fixing-list|
+{{ track_fixing_list }}
 
 You can then type the number key listed next to the track (while still holding down the control or command key) to assign the selected instance to the corresponding track. In the image above, you'd want to type **command-1** to assign the orange instance to the red "F" track.
 
-
 **Mistaken Identities:** The code may misidentify which instance goes in
 which track.
 
-Mistake identities are harder to correct since there's no certain way to find them—if we knew where they were, then we wouldn't have gotten them wrong in the first place. But there are some strategies to make it easier to locate them in your predictions.
+Mistaken identities are harder to correct since there's no certain way to find them—if we knew where they were, then we wouldn't have gotten them wrong in the first place. But there are some strategies to make it easier to locate them in your predictions.
 
 One strategy is to set the trail length to **50** and jump through the predictions 50 frames at a time using the **down arrow** key. It's usually possible to see identity swaps by looking at the shape of the track trails, as here:
```

Comment:
Refer to the GUI shortcut names instead of the current default key (also do not refer to a specific step size, just refer to the shortcut name and the general description) - for easier maintenance down the road.